src/app/pages/task-calendar/replication/replication-form/replication-form.component.ts

```diff
@@ -348,9 +348,11 @@ export class ReplicationFormComponent implements AfterViewInit {
         type : 'textareabutton',
         name : 'repl_remote_hostkey',
         placeholder : T('Remote Hostkey'),
-        tooltip : T('Use <b>Scan SSH Key</b> to retrieve the public host\
-                     key of the remote system and enter or paste the key\
-                     here.'),
+        tooltip : T('Paste the host key of the remote system configured\
```

Comment:
s/remote system configured/destination NAS/

Reply:
Fixed

I left 'remote system' thinking that any remote system could be set up as a replication target, not just a NAS.